app/uploaders/lighthouse_supporting_evidence_attachment_uploader.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# Files uploaded as part of a form526 submission that will be sent to Lighthouse upon form submission.
+class LighthouseSupportingEvidenceAttachmentUploader < LighthouseDocumentUploaderBase
+  extend Logging::ThirdPartyTransaction::MethodWrapper
+
+  wrap_with_logging(
+    :store,
+    additional_instance_logs: { upload_file: %i[size headers] }
+  )
+
+  def initialize(guid, _unused = nil)
+    # carrierwave allows only 2 arguments, which they will pass onto
+    # different versions by calling the initialize function again
+    # so the _unused argument is necessary
+
+    super
+    @guid = guid
+
+    #  defaults to CarrierWave::Storage::File if not AWS
+    if Rails.env.production?
+      set_aws_config(
+        Settings.evss.s3.aws_access_key_id,
+        Settings.evss.s3.aws_secret_access_key,
+        Settings.evss.s3.region,
+        Settings.evss.s3.bucket
+      )
+    end
+  end
+
+  def store_dir
+    raise 'missing guid' if @guid.blank?
+
+    "disability_compensation_supporting_form/#{@guid}"
```

Comment:
Are Lighthouse uploads stored to the same directory as EVSS supporting evidence attachments? Is the `guid` for each type unique?